cdc/log.cc

```diff
@@ -880,12 +881,24 @@ class log_mutation_builder {
         return _base_schema;
     }
 
+    clustering_key create_ck(int batch) const {
+        auto log_ck = clustering_key::from_exploded(_log_schema, { _tuuid, int32_type->decompose(batch) });
+        set_key_columns(log_ck, _base_schema.partition_key_columns(), _base_pk);
```

Comment:
this is definitely not `const`.
I see what you wanted to do here, leme suggest the following: move the `set_key_columns` call to `allocate_new_log_row`. Then `create_ck` is indeed `const` (it's just a calculation), and `set_key_columns` doesn't need to be `const`, as it should not be.

Reply:
true.

Reply:
Fixed.